cou/commands.py

```diff
@@ -83,8 +83,7 @@ def get_subcommand_common_opts_parser() -> argparse.ArgumentParser:
         help="Set the model to operate on.\nIf unset, the model name will be determined by "
         "inspecting the environment as follows:\n"
         "  1 - Environment variable JUJU_MODEL,"
```

Comment:
`JUJU_MODEL` it's  not been used on [pythonlib-juju](https://github.com/juju/python-libjuju/blob/2.9/juju/client/jujudata.py#L35). I've manually tested and it does not work. I think we should remove it too and let just the `Current active juju model`

Reply:
True, I did not know that it's not working since I'm using only `JUJU_DATA`.